scripts/leetcode_directory_md.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+from urllib.request import Request, urlopen

```

Comment:
This library is a defacto standard for python client side stuff :smile:
```suggestion
import requests
```